### ISSUE TITLE
[Package Request] python3-httpx

### ISSUE THREAD

**Issue by imrn** [void-linux/void-packages#22033]
A next-generation HTTP client similar to python3-requests.

HTTPX is a fully featured HTTP client for Python 3, which provides sync and async APIs, and support for both HTTP/1.1 and HTTP/2.

https://www.python-httpx.org
https://github.com/encode/httpx
```
Dependencies
    urllib3 - Sync client support.
    h11 - HTTP/1.1 support.
    h2 - HTTP/2 support.
    certifi - SSL certificates.
    chardet - Fallback auto-detection for response encoding.
    hstspreload - determines whether IDNA-encoded host should be only accessed via HTTPS.
    idna - Internationalized domain name support.
    rfc3986 - URL parsing & normalization.
    sniffio - Async library autodetection.
    brotlipy - Decoding for "brotli" compressed responses. (Optional)
```

Thanks in advance..

**Comment by misuchiru03** [comment]
Don't know how well this will work. [async 0.6.2](https://pypi.org/project/async/) is reported as not functioning in python3 and its major user is not being patched.  Attempting it, and if anyone has a use for it afterward, test it out.

**Comment by imrn** [comment]
?? I don't think async package is needed. Why do you think so?

**Comment by misuchiru03** [comment]
You are correct. I missed a line in sniffio's setup.  Correcting.

**Comment by abenson** [comment]
Closed in 926521e685f4b36d7699684942faf2da3b720bfc
